src/Modules/UngradedModules/FreeResolutions.jl

```diff
@@ -570,22 +572,44 @@ function free_resolution_via_kernels(M::SubquoModule, limit::Int = -1)
   end
   C = Hecke.ComplexOfMorphisms(ModuleFP, mp, check = false, seed = -2)
   #set_attribute!(C, :show => free_show, :free_res => M) # doesn't work
+  set_attribute!(C, :show => Hecke.pres_show, :free_res => M)
   return FreeResolution(C)
 end
 
 @doc raw"""
-    free_resolution(I::MPolyIdeal; length::Int=0, algorithm::Symbol=:fres)
+    free_resolution(I::MPolyIdeal; length::Int = 0, algorithm::Symbol = :fres)
 
-Compute a free resolution of `I`.
+Consider `I` as a subquotient module over its base ring, and return a free resolution of that module.
 
 If `length != 0`, the free resolution is only computed up to the `length`-th free module.
-At the moment, options for `algorithm` are `:fres`, `:mres` and `:nres`. With `:mres` or `:nres`,
-minimal free resolutions are returned.
+At the moment, options for `algorithm` are `:fres`, `nres`, and `:mres`. With `:mres`,
+in the positively graded case, a minimal free resolution is returned.
```

Comment:
What about :mres in the local case? In the Singular backend, this is minimal as well. I assume that this is either not available in Oscar yet or documented elsewhere?

Reply:
This requires more functionality on the module side and is not part off this PR.

Reply:
Fine with me!